src/views/mfa-verify/SendEmailAndVerifyCodeForm.js

```diff
@@ -60,8 +60,8 @@ const SendEmailAndVerifyCodeFormVerifyEmailCodeForm = Form.extend({
   initialize: function () {
     Form.prototype.initialize.apply(this, arguments);
 
-    //Added thorttle to prevent keyboard enter trigger multipele API calls
-    this.handleSubmit = _.throttle(this.handleSubmit, 100, { leading: false });
+    // Added throttle to prevent keyboard enter trigger multiple API calls
+    this.handleSubmit = _.throttle(this.handleSubmit, 200, { leading: false });
```

Comment:
is it going to work on slow internet connection? Also want to get some insight how increasing the throttle works and won't break in future or with varied internet bandwidths

Reply:
I'm also wondering the same